.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,64 @@
+# To enable this workflow on a fork, comment out:
+#
+# if: github.repository == 'numpy/numpy'
```

Comment:
I guess this line should **not** be commented out

Reply:
It's a confusing comment, but it is the same as the comments in all of the other workflows. It's referring to line 15.

Reply:
Ah OK makes sense ...